doc/contrib/doc.rst

```diff
@@ -0,0 +1,38 @@
+.. _contribdoc:
+
+Writing documentation
+=====================
+
+The source for `the documentation <https://arbor.readthedocs.io>`__ is
+found in the ``/doc`` subdirectory, C++ examples in ``/example`` and
+Python examples in ``/python/example``. You can add yours in the same
+way you would contribute code, please see the “Github workflow” section.
+
+.. todo::
+    Very much in progress.
```

Comment:
What is a work in progress?
If you are referring to the documentation, maybe a statement like `The documentation in Arbor is currently a work in progress`.